DVCLive: Deprecate `live` section of `dvc.yaml`. (#3411)
iterative/dvc.org: content/docs/dvclive/get-started.md

See `Live.log()`, `Live.log_image()` and `Live.log_plot()` for more details.

### HTML report

If and when `step` is updated, DVCLive generates or updates an HTML report in

> `dvclive/report.html` which will contain all the logged data.

Reviewer: Is this only true if `report="html"` was passed to `Live()` (per `set/next_step()` refs update)?

If so do we need to specify so [in the Get Started](https://dvc.org/doc/dvclive/get-started#initialize-dvclive) or is it the default?

Reviewer: > Is this only true if report="html" was passed... or is it the default?

I see it's the default. No further change needed in GS then 👍🏼

See `Live.log()`, `Live.log_image()` and `Live.log_plot()` for more details.

### HTML report

> By default, DVCLive generates an HTML report in `dvclive/report.html`.
> 
> This report will contain all the logged data and will be automatically updated
> during training on each `step` update!

Reviewer: Quick clarification
```suggestion
If and when `step` is updated, DVCLive generates or updates an HTML report in
`dvclive/report.html` which will contain all the logged data.
```

> Will commit to see if that triggers a review app build...
